babel.config.js

```diff
@@ -1,24 +1,5 @@
 const path = require('path');
```

Comment:
I synced this file with the core one, minus some things (we don't have there error management for instance).
I will do some tests for the packages still on the old bundling. But I would advise against releasing a state where some package use the core bundling strategy and others don't.